CMakeLists.txt

```diff
@@ -131,6 +131,87 @@ endif()
 
 # -----
 
+if (DEFINED TRITON_PLUGIN_DIRS)
```

Comment:
I verified using `git diff -w` that this is just code motion.  Wish github showed it more clearly.  Anyway LGTM.